src/calendar/get_available_time_slots.rs

```diff
@@ -0,0 +1,82 @@
+use crate::routes::get_service_duration_by_id;
+use actix_web::web;
+use chrono::{DateTime, Duration, Utc};
+use google_calendar3::{
+    api::{FreeBusyRequest, FreeBusyRequestItem},
+    hyper, hyper_rustls, oauth2,
+    oauth2::ServiceAccountAuthenticator,
+    CalendarHub,
+};
+use log::{error, info};
+use sqlx::PgPool;
+use std::default::Default;
+use std::error::Error;
+
+const SERVICE_ACCOUNT_KEY_FILE: &str = "calendar-json.googleapis.com.json";
+const CALENDAR_ID: &str = "bbbb.iam.gserviceaccount.com";
```

Comment:
this information should be placed in the env vars instead of the str literal in the code

Reply:
I totally agree for security it is better to add in envs of course. In this form I did it more for clarity.